src/Components/SceneList/Internal/SceneDialog.tsx

```diff
@@ -238,7 +260,10 @@ const SceneDialog: React.FC<ISceneDialogProps> = ({
                 selectedSceneCopy.latitude = lat;
```

Comment:
we should not need the forked logic for `sceneToEdit` everything should go through the reducer.